src/cmds.c

```diff
@@ -136,30 +136,6 @@ go to its beginning.  */)
   return make_number (count <= 0 ? - shortage : shortage);
 }
 
-DEFUN ("beginning-of-line", Fbeginning_of_line, Sbeginning_of_line, 0, 1, "^p",
-       doc: /* Move point to beginning of current line (in the logical order).
-With argument N not nil or 1, move forward N - 1 lines first.
-If point reaches the beginning or end of buffer, it stops there.
-
-This function constrains point to the current field unless this moves
```

Comment:
Just need to add this second paragraph to the docstring, and we're good to go :)

Reply:
Done. Are we ready to merge?

Reply:
We have to wait for #456.